src/libraries/System.Private.CoreLib/src/Resources/Strings.resx

```diff
@@ -3299,8 +3305,8 @@
   <data name="Resources_StreamNotValid" xml:space="preserve">
     <value>Stream is not a valid resource file.</value>
   </data>
-  <data name="RFLCT_AmbigCust" xml:space="preserve">
```

Comment:
@buyaa-n  do you know the answer to this?

Reply:
Not sure if I understood the question, `xml:space="preserve"` is for preserving the whitespaces in the string not the string. Even if the given string doesn't have whitespaces that need to be preserved it could be useful in case the message updated with a string that have a carriage return. For more info: https://learn.microsoft.com/en-us/dotnet/desktop/xaml-services/white-space-processing#white-space-in-inner-text-and-string-primitives

Reply:
My question was about me _removing_ a string from the resx file? Is that okay (since will not use this anymore) or should I just leave it in the resx as an orphan?

Reply:
Ah sorry for misunderstanding, unused strings should be removed